Reverse-Proxy-with-Cloudflare-Tunnel.md

```diff
@@ -92,3 +92,16 @@ With this approach, you don't even need to expose your container port to the hos
 
 UPTIME_KUMA_CLOUDFLARED_TOKEN=`<your token>`
 
+
+## Troubleshooting
+### How to tunnel multiple applications
+
+When you start a Cloudflare tunnel through an Uptime Kuma docker install, that tunnel can only be used for Uptime Kuma.
+Running duplicate tunnels is [not supported](https://developers.cloudflare.com/cloudflare-one/faq/cloudflare-tunnels-faq/#how-can-i-troubleshoot-a-tunnel-that-was-configured-from-zero-trust) and might result in unstable connections.
+
+If you would like to proxy other containers or services through Cloudflare, then you will need to deactivate the tunnel that's built into Uptime Kuma, and install the cloudflared connector manually.
+Specific instructions to install the cloudflared connector should be available on the same page that you copied the token from eariler. 
+
+Installing the connector directly will allow you to tunnel Uptime Kuma alongside other applications by adding additional Public Hostnames on the dashboard.
```

Comment:
This is a very long 3 paragraphs for:
> When starting a `Cloudflare Tunnel` through `uptime-kuma`, this tunnel is limited to Uptime Kuma, as running duplicate tunnels is unsupported.
> If you like to use the `Cloudflare Tunnel` for other applications, refer to the `cloudflared` documetation and use it instead of the build-in connecter.

I know that a bit of fluff is good in documentations, but I think this is a bit excessive. What do you think about this?

Reply:
Sounds great. I like your phrasing better.

Reply:
```suggestion
When starting a `Cloudflare Tunnel` through `uptime-kuma`, this tunnel is limited to Uptime Kuma, as running duplicate tunnels is unsupported.

If you like to use the `Cloudflare Tunnel` for other applications, refer to the `cloudflared` documetation and use it instead of the build-in connector.
Specific instructions to install the cloudflared connector are available on the same page that you copied the token from eariler.
```